diff-containers/changelog.d/20230208_173935_joris_readmes.md

```diff
@@ -0,0 +1,20 @@
+First release
+
+<!--
+### Patch
+
+- A bullet item for the Patch category.
+
+-->
+<!--
+### Non-Breaking
+
+- A bullet item for the Non-Breaking category.
+
+-->
+<!--
+### Breaking
+
+- A bullet item for the Breaking category.
+
+-->
```

Comment:
I'm not sure what this entry is about? I don't think these first files should be here.

Reply:
I'll remove them -- I can add "first release" changelog entries when the actual publishing happens